thesis-api-v2/src/main/resources/search_phrases.txt

```diff
@@ -0,0 +1,12 @@
+name;search_word;quantity
+animals;types of mammals;100
+animals;types of lizards;100
+animals;types of birds;100
+plants;types of herbs;150
+plants;types of flowers;150
+plants;types of trees;100
+geography;mount everest;150
+geography;amazon river;150
+geography;luzon;150
+parts of human body;parts of human body;90
+cars;types of cars;100
```

Comment:
I would suggest adding some `search_words` that were before in `categories.txt` such as `types of animals` and `types of plants`. Sure, we will lose clear division into subcategories, but we should focus on getting as many good terms for `categories` as possible. Sure, there will be duplicates between subcategories (but we are already filtering them) and we will need to ge rid of one of the condition for getting wrong answers (details in one of the comments), but In my opinion we should focus on main `categories` and use this `search_words` to get as many good terms for them as possible.